include/swift/ABI/MetadataValues.h

```diff
@@ -747,7 +749,8 @@ class TargetFunctionTypeFlags {
     ParamFlagsMask    = 0x02000000U,
     EscapingMask      = 0x04000000U,
     // SWIFT_ENABLE_TENSORFLOW
-    DifferentiableMask  = 0x08000000U
+    DifferentiableMask  = 0x08000000U,
```

Comment:
I tried using a single `DifferentiabilityMask`, similar to the one in `AnyFunctionType::ExtInfo`:
```cpp
      DifferentiabilityMask  = 0xF8000000U,
      DifferentiabilityShift = 24U
```

But I didn't figure out how to make it work all the way. Things are tricky because the mask is `F8` instead of `FF`, which requires extra bitshifting.